src/builders/anndata_builders.py

```diff
@@ -34,11 +36,20 @@ def get_conf_cells(self):
         # or AnnData default (like X_umap or X).
         cell_set_obs = ["leiden"]
         cell_set_obs_names = ["Leiden"]
-        dags = [dag
-                for dag in self._entity['metadata']['dag_provenance_list'] if 'name' in dag]
+        dags = [
+            dag for dag in self._entity['metadata']['dag_provenance_list']
+            if 'name' in dag]
         if(any(['azimuth-annotate' in dag['origin'] for dag in dags])):
-            cell_set_obs.append("predicted.ASCT.celltype")
-            cell_set_obs_names.append("Predicted ASCT Cell Type")
+            response = requests.get(
+                f'{adata_url}/uns/annotation_metadata/is_annotated/0',
+                headers=self._get_request_init()['headers'])
+            if response.ok:
```

Comment:
We need to check the actual value returned.  This isn't just about its presence but whether or not the returned bytes are 1 or 0

Reply:
So checking `response.content == b'\x01'` is probably what we want

Reply:
will do: thanks!